computing_environment/forms/custom_signup.py

```diff
@@ -15,15 +15,15 @@ class CustomSignupForm(SignupForm):
         )
     )
 
-    first_name = forms.CharField(max_length=30, label='First Name', 
+    first_name = forms.CharField(max_length=50, label='First Name', required=False,
```

Comment:
Na pewno ma być required=False, czy to przechodzi potem przez jakąś inną walidacje?

Reply:
To dotyczy tylko formularza. Walidacja jest w modelu. Bez tego przeglądarka nie pozwoli zatwierdzić pustego pola.